src/Converter/Feature2Mesh.js

```diff
@@ -484,6 +484,8 @@ export default {
 
             group.quaternion.copy(collection.quaternion);
             group.position.copy(collection.position);
+            group.scale.copy(collection.scale);
+            group.feature = collection;
```

Comment:
why the collection is referenced in group.feature?

Reply:
removed